[2022-05-13 15:28:19,276] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: my_dag_v0.0.1.my_task_3 2022-05-04T00:00:00+00:00 [queued]>
[2022-05-13 15:28:19,305] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: my_dag_v0.0.1.my_task_3 2022-05-04T00:00:00+00:00 [queued]>
[2022-05-13 15:28:19,305] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 15:28:19,305] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-05-13 15:28:19,307] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 15:28:19,314] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): my_task_3> on 2022-05-04T00:00:00+00:00
[2022-05-13 15:28:19,321] {standard_task_runner.py:52} INFO - Started process 398 to run task
[2022-05-13 15:28:19,323] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'my_dag_v0.0.1', 'my_task_3', '2022-05-04T00:00:00+00:00', '--job-id', '35', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag_v_0_1_0_0.py', '--cfg-path', '/tmp/tmpkbb24v1a', '--error-file', '/tmp/tmp4228f7by']
[2022-05-13 15:28:19,325] {standard_task_runner.py:77} INFO - Job 35: Subtask my_task_3
[2022-05-13 15:28:19,354] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag_v0.0.1.my_task_3 2022-05-04T00:00:00+00:00 [running]> on host bacb497a28fb
[2022-05-13 15:28:19,395] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=msbeni@gmail.com
AIRFLOW_CTX_DAG_OWNER=msbeni
AIRFLOW_CTX_DAG_ID=my_dag_v0.0.1
AIRFLOW_CTX_TASK_ID=my_task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-05-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-04T00:00:00+00:00
[2022-05-13 15:28:19,396] {python.py:151} INFO - Done. Returned value was: None
[2022-05-13 15:28:19,403] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=my_dag_v0.0.1, task_id=my_task_3, execution_date=20220504T000000, start_date=20220513T152819, end_date=20220513T152819
[2022-05-13 15:28:19,438] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-13 15:28:19,455] {local_task_job.py:151} INFO - Task exited with return code 0
